ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-ec.yaml

```diff
@@ -592,7 +592,6 @@ tests:
   steps:
     cluster_profile: azure-qe
     env:
-      ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI
```

Comment:
Since https://github.com/openshift/release/pull/42920 get merged, many jobs are removed from ec ci, so I think we do not update this file any more.

Reply:
acked, will update

Reply:
updated, job `azure-ipi-baselinecaps-v412-to-multiarch-f14` seems in this yaml file, remove `ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI` from its env.